packages/core-data/src/resolvers.js

```diff
@@ -453,7 +453,7 @@ __experimentalGetTemplateForLink.shouldInvalidate = ( action ) => {
 	);
 };
 
-export const __experimentalGetCurrentGlobalStylesId =
+export const __experimentalGetCurrentGlobalStyles =
```

Comment:
The reason why I'm returning the entire global style object, instead of just the ID, is so we can access the [revisions url.](https://github.com/WordPress/gutenberg/pull/46667/files#diff-fec6390e24332dec4a0c7c74f9f7b9a015ce191e0e6348361ead793ceda3991dR509) 

E.g.: `currentGlobalStyles?._links?.[ 'version-history' ]?.[ 0 ]?.href;`

Reply:
It seemed a waste not to repurpose this method since it was fetching what we wanted anyway and only returning the id. 

Plus it's experimental so I didn't see much harm in changing it.